README.md

```diff
@@ -69,6 +75,33 @@ residual_err = np.array(x_err) ^ np.array(result.estimate)
 print(code.is_x_stabilizer(residual_err))
 ```
 
+### Example for applying error correction to a circuit
+
+```python3
+from mqt import qecc
+
+file = "path/to/qasm/file.qasm"  # Path to the openqasm file the quantum circuit shall be loaded from
+ecc = "Q7Steane"  # Error correction code that shall be applied to the quantum circuit
+ecc_frequency = 100  # After how many times a qubit is used, error correction is applied
+
+result = qecc.apply_ecc(file, ecc, ecc_frequency)
+
+# print the resulting circuit
+print(result["circ"])
+```
+
+We provide a wrapper script for applying error correction to quantum circuits (provided as openQasm) and followed by a
```

Comment:
The proper way of writing the format is OpenQASM.

Reply:
Changed this